tachyons-to-tailwind-transform.js

```diff
@@ -0,0 +1,319 @@
+// Installation:
```

Comment:
this does a codemod on react files that are tachyons and tries to make them tailwind `yarn codemod:tachyons-to-tailwind /path`